### PR TITLE
Fix Parameter Names of get_resource_path and get_resource_content

### PR DESCRIPTION
**Pull request by srkunze**
and fixed some documentation, error messages and newlines.

I hope everything plays out as desired.

### CODE REVIEW

**Comment by srkunze** [comment]
Could somebody check that I used the `.. versionadded:: Boron` tag for parameters correctly?

File: salt/modules/virtualenv_mod.py [hunk 1]
```diff
@@ -335,16 +333,22 @@ def get_distribution_path(venv, distribution):
     return ret['stdout']
 
 
-def get_resource_path(venv, package_or_requirement, resource_name):
+def get_resource_path(venv, package=None, resource=None, package_or_requirement=None, resource_name=None):
```
Comment (jfindlay) [inline salt/modules/virtualenv_mod.py]: @srkunze, will you move these new args to the end of the list to minimize API change?  Thanks.

File: salt/modules/virtualenv_mod.py [hunk 2]
```diff
@@ -369,17 +384,22 @@ def get_resource_path(venv, package_or_requirement, resource_name):
     return ret['stdout']
 
 
-def get_resource_content(venv, package_or_requirement, resource_name):
+def get_resource_content(venv, package=None, resource=None, package_or_requirement=None, resource_name=None):
```
Comment (jfindlay) [inline salt/modules/virtualenv_mod.py]: @srkunze, same here.

**Comment by srkunze** [comment]
Done. :)

**Comment by jfindlay** [comment]
Thanks, @srkunze.

**Pull request merged**